FreeSurfer/custom/fs_to_gii

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+if [ -z "$2" ]; then
+	echo "usage: fs_to_gii <freesurfer dir> <in> <out gii>"
+	exit -1
+fi
+
+FreeSurferFolder=$1
+in=$2
+out=$3
+
+set -x
+mris_convert $in $out.temp.gii
+set +x
+
+MatrixXYZ=`mri_info --cras ${FreeSurferFolder}/mri/brain.finalsurfs.mgz`
```

Comment:
This looks like the legacy c_ras code, in which case simply doing `mris_convert --to-scanner` and nothing else is probably the better solution.

Reply:
Yes that should be used if we don't care about supporting old versions that don't have it.  I don't think 5.3 had it.

Reply:
All my tests ran on version 6. Version 5 is not currently supported so I'm following Tim's recommendation.